src/pages/Home/components/BookSlider.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import Slider from "react-slick";
+import PropTypes from 'prop-types';
+import "./BookSlider.css";
+
+const SampleNextArrow = (props) => {
+  const { className, style, onClick } = props;
+  return (
+    <div
+      className={`${className} custom-arrow custom-next-arrow`}
+      style={{ ...style, display: 'block' }}
+      onClick={onClick}
+    />
+  );
+}
+
+const SamplePrevArrow = (props) => {
+  const { className, style, onClick } = props;
+  return (
+    <div
+      className={`${className} custom-arrow custom-prev-arrow`}
+      style={{ ...style, display: 'block' }}
+      onClick={onClick}
+    />
+  );
+}
+
+const Book = ({ imageUrl }) => (
```

Comment:
Same issues as last time. Please make this its own component with a different name.